### PR TITLE
[1678] Added back link to guidance page

### PR DESCRIPTION
**Pull request by markmead**
### Context

Add a button to return to the referrer page so that the page no longer feels like a dead end.

### Changes proposed in this pull request

Button to return back to the page the user came from no matter where they are.

### Guidance to review

Should this become a component/partial that we can reuse on other pages? Such as cookies, terms etc.

### CODE REVIEW

**Comment by markmead** [comment]
[Add 'back' link to publishing guidance page](https://trello.com/c/rDEirWaq/1678-add-back-link-to-publishing-guidance-page)

File: app/views/pages/guidance.html.erb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <%= content_for :page_title, "Guidance" %>
 
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(:back) %>
```
Comment (dankmitchell) [inline app/views/pages/guidance.html.erb]: What if the `referrer` is nil?

I think it should take them back to `provider#index` which will force them to sign in...

Comment (fofr) [reply app/views/pages/guidance.html.erb]: If it's nil I think it defaults to `history.back`

Comment (markmead) [reply app/views/pages/guidance.html.erb]: Gonna test this now.

Comment (fofr) [reply app/views/pages/guidance.html.erb]: What happens to `providers_path` if the user isn't logged in?

Comment (dankmitchell) [reply app/views/pages/guidance.html.erb]: The get taken to DfE-Sign

Comment (fofr) [reply app/views/pages/guidance.html.erb]: What happens if the user has only 1 organisation?

Comment (dankmitchell) [reply app/views/pages/guidance.html.erb]: They are re-directed to the `org#show` view

**Pull request merged**